manga_auto_translator/cli_args.py

```diff
@@ -0,0 +1,8 @@
+from argparse import ArgumentParser

```

Comment:
(OPTIONAL) FYI : ArgumentParser is fine, if you want to make clean client script, you can use click package which is better. -> https://click.palletsprojects.com/en/8.1.x/